### PR TITLE
Update app-protection-policy-settings-android.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the document with Box as it is now available as an option to Allow user to save copies to selected services'

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by PRMerger18** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/memdocs/issues/2144

**Comment by yogkumgit** [comment]
@Erikre Please merge this PR. Thanks

File: memdocs/intune/apps/app-protection-policy-settings-android.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ There are three categories of policy settings: data protection settings, access
...
-|<ul><b><ul><b>**Allow user to save copies to selected services** |Users can save to the selected services (OneDrive for Business, SharePoint, and Local Storage). All other services will be blocked.  | **0 selected** |
+|<ul><b><ul><b>**Allow user to save copies to selected services** |Users can save to the selected services (OneDrive for Business, SharePoint, Box and Local Storage). All other services will be blocked.  | **0 selected** |
```
Comment (JohanFreelancer9) [inline memdocs/intune/apps/app-protection-policy-settings-android.md:51]:
```suggestion
|<ul><b><ul><b>**Allow user to save copies to selected services** |Users can save to the selected services (OneDrive for Business, SharePoint, Box, and Local Storage). All other services will be blocked.  | **0 selected** |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by PRMerger15** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**